### ISSUE TITLE
libncursesw.5.dylib Library not loaded using Python 3.10.7 on Mac OS 12.6

### ISSUE THREAD

**Issue by edmit819** [marcelotduarte/cx_Freeze#1624]
I started receiving the below error message on execution of my application once I built my project with Python 3.10.7 on Mac OS 12.6 using cx_Freeze 6.11.1
ImportError: dlopen(/Volumes/DevDrive/Data/bin/Console-Mac.app/lib/_curses.cpython-310-darwin.so, 0x0002): Library not loaded: '@loader_path/../lib/libncursesw.5.dylib'

Any insights would be greatly appreciated on how to resolve this issue.

**Comment by marcelotduarte** [comment]
> I started receiving

Do you get success in a previous version?

**Comment by edmit819** [comment]
Yes, I was able to build and execute cx_Freeze 6.11.1 on Python 3.9.14.  Not sure why I'm having an issue on 3.10.7, I take it no one else is having this issue? [I stand corrected, I was able to execute cx_Freeze 6.9 on Python 3.9.14, not cx-Freeze 6.11.1]

**Comment by marcelotduarte** [comment]
> I take it no one else is having this issue?

I think no one else.

Try to use:
`pip install --no-binary=cx_Freeze cx_Freeze`

**Comment by edmit819** [comment]
Well, I got everything now to work using Python 3.10.7.  I noticed when I upgraded to Python 3.10 that it updated setuptools to version 65.  cx-Freeze uses setuptools version 60.10.0.  I had to uninstall setuptools and cx-Freeze.  When I reinstall cx-Freeze, it updated it to 6.11.1 and setuptools to version 60.10.0.  When I uninstalled cx-Freeze and reinstalled  cx-Freeze version 6.9 everything seems to work fine now...  What a journey...

However, not sure why cx-Freeze 6.11.1 wouldn't work...No, I did not get a chance to run the command: _pip install --no-binary=cx_Freeze cx_Freeze_.  Have you tested cx-Freeze 6.11.1 on Mac OS 12.6?
